### PR TITLE
Fix double args

### PR DESCRIPTION
**Pull request by dkropachev**
https://trello.com/c/EPQYaUeU/3537-investigate-enterprise-version-running-on-sct-master-failure

When scylla help mentions any variable twice, scylla node fails to start.
This fix will publish error event instead.

TIme to time enterprise scylla have doubled help message and it blocks us from running tests on it.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by bentsi** [comment]
this is a functional test, we need to do it in dtest. Any reason to check this in SCT?

**Comment by dkropachev** [comment]
> this is a functional test, we need to do it in dtest. Any reason to check this in SCT?

It is not a test, currently sct will fail on that, in particular if you try to run sct master against 2021.1 it will fail on node setup because of double args.
Idea of the fix not to break a test, just fail it.

**Comment by bentsi** [comment]
why the nide setup will fail on double args?

**Comment by bentsi** [comment]
please add an event also to ScyllaArgParser.filter_args method

**Comment by dkropachev** [comment]
> please add an event also to ScyllaArgParser.filter_args method

Done.

**Comment by dkropachev** [comment]
Rebased to the master and fixed pre-commit problems

File: defaults/severities.yaml [hunk 1]
```diff
@@ -9,6 +9,8 @@ DataValidatorEvent.DataValidator: ERROR
 DataValidatorEvent.ImmutableRowsValidator: ERROR
 DataValidatorEvent.UpdatedRowsValidator: WARNING
 DataValidatorEvent.DeletedRowsValidator: ERROR
+ScyllaHelpErrorEvent.duplicate: ERROR
+ScyllaHelpErrorEvent.filtered: NORMAL
```
Comment (bentsi) [inline defaults/severities.yaml:13]: Let's make it error by default

**Review by bentsi — APPROVED** [review]

**Review by bentsi — APPROVED** [review]

**Pull request merged**